lib/funky/page.rb

```diff
@@ -0,0 +1,41 @@
+module Funky
+  class Page < FunkyObject
```

Comment:
@bruce-fs Or you could even add the `@see` comment here and describe what a Page is (still from the Facebook docs)